src/features/content-explorer/content-explorer/ContentExplorer.scss

```diff
@@ -71,6 +71,33 @@
         }
     }
 
+    .bdl-ContentExplorerIncludeSubfolders {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        height: 10 * $bdl-grid-unit;
+        margin-top: 20px;
+        margin-bottom: -10px;
+
+        .bdl-ContentExplorerIncludeSubfolders-toggleIconContainer {
```

Comment:
why is this div necessary? shouldn't the `display: flex;` and `align-items: center;` on the outer div be enough to center everything horizontally including the checkbox?

Reply:
This was to wrap the info badge icon and toggle together so I can do justify content space between with this div and the checkbox. 
I can alternatively take this div out then margin auto the checkbox to the right

Reply:
you can leave this for now but I think it might be better to put the info tooltip inside the label prop of the tooltip similar to this component: https://github.com/box/box-ui-elements/blob/master/src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx#L33-L55

Reply:
Ah I see that is a good idea. I would have to wrap the FormattedMessage component and tooltip tg to pass as a label prop, but it would be cleaner